pkg/providers/aws/instance_topology.go

```diff
@@ -184,7 +186,7 @@ func toGraph(topology []types.InstanceTopology, cis []common.ComputeInstances) (
 				ID:   instanceID,
 			}
 		}
-		forest["cpu-nodes"] = sw
+		forest["no-topology"] = sw
```

Comment:
So is this "no-topology" vertex going to be a hard-coded switch for tree topology? In that case, should we placed it in the consts file in common and edit some of the tests (such as in grpc_client_test.go) to follow the same strucutre?